components/nexus-blobstore-api/src/main/java/org/sonatype/nexus/blobstore/api/BlobStore.java

```diff
@@ -120,4 +120,19 @@
    * @return Iterable handle must be closed when finished using it.
    */
   AutoClosableIterable<BlobId> iterator();
+
+  /**
+   * @return true if this blob store is is in use
+   */
+  boolean isInUse();
+
+  /**
+   * Increases the usage of this blob store;
```

Comment:
I thought this was some instruction to use the blob store more, or less. Perhaps:
"Increments this blob store's the usage counter"